project-list.md

```diff
@@ -0,0 +1,133 @@
+# Active Projects for [nteract](https://github.com/nteract)
+
+Date updated: 2018-10-23
+
+Productivity and teamwork | Projects | Dev Tools and Utilities | Archived
+
+---
+
+## Productivity and teamwork
+
+### Project Management and Planning
+
+- [meeting-minutes](https://github.com/nteract/meeting-minutes) and (google doc)
+- *roadmap* (google doc)
+- specs
+
+
+### Marketing, Communications, and Public Relations
+
+- assets
+- design
+- logos
+- nteract-monthly-newsletter
+- [nteract.io](https://github.com/nteract/nteract.io): website
+
+
+### Documentation
+
+- desktop-manual
+- [docs](https://github.com/nteract/docs)
+- docs-old
+- education
+
+
+---
+
+## Project Areas
+
+### commutable
+
+- commutable
+- commutable-models
+- commutable-perf
+
+- [commuter-on-glitch](https://github.com/nteract/commuter-on-glitch)
+
+### enchannel
+
+- enchannel
+- enchannel-socketio-backend
+- enchannel-notebook-backend
+- enchannel-in-memory
+
+### hydrogen: notebooks in atom editor
+
+- [hydrogen](https://github.com/nteract/hydrogen)
+
+
+### nteract: desktop
+
+- [nteract](https://github.com/nteract/nteract)
+- desktop-integration-tests
+
+
+### vizualization
+
+- [vega-embed-v2](https://github.com/nteract/vega-embed-v2)
+
+## Workflow
+
+### papermill: workflows for notebook execution
+
+- [papermill](https://github.com/nteract/papermill)
+- papermillr
+- [scrapbook](https://github.com/nteract/scrapbook)
+
+### bookstore: workflows for notebook storage
+
+- [bookstore](https://github.com/nteract/bookstore)
+
+
+## Dev Tools and Utilities
+
+### Utilities - development
+
+- jupyter-paths
+- ipython-paths
+
+
+# Active or inactive
+
+- ion
+- mathjax-electron
+- galleria
+- zmq-prebuilt-testing
+- create-nteract-app
+- minimal-plotly
+- dx
```

Comment:
Likely going to be putting more cycles into this as we carve out a spec for data explorer metadata